qgis.bash

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -l
+
+function maximize_qgis_window() {
+    # Set the desired size
+    desired_width=1680
+    desired_height=999 # or 1050
```

Comment:
@Rub21 shouldn't we be getting these values based on the actual width and height values of the user's screen instead of hard-coding?

Reply:
sure, i fixed it,